.github/workflows/windows.yml

```diff
@@ -0,0 +1,56 @@
+name: windows
+
+on:
+  push:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'
+  pull_request:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'
+
+jobs:
+  windows-msvc:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install Meson and Ninja
+      run: pip install meson ninja

```

Comment:
ninja is a c++ project, so it should not be available in pip. My guess is that it's available in the default image? if so, it can be also dropped in the mingw case below

Reply:
Unfortunately the install step for meson/ninja is needed in both msvc/mingw

Reply:
meson is one thing ninja is another.

`pip install ninja` does not work here, which is to be expected since it's not a python module. Have you tried changing the above to "pip install meson" and dropping "ninja" from the mingw/pacboy section - what kind of error do you see in the CI?

Reply:
You're correct, I misread the first comment and removed both meson/ninja and it failed. But just leaving meson works correctly, did that change now.